scss/_utilities.scss

```diff
@@ -602,14 +615,15 @@ $utilities: map-merge(
       values: map-merge(
         $utilities-bg-colors,
         (
-          "body": $body-bg,
-          "white": $white,
           "transparent": transparent,
-          "supporting-green": $supporting-green,
-          "supporting-blue": $supporting-blue,
-          "supporting-yellow": $supporting-yellow,
-          "supporting-pink": $supporting-pink,
-          "supporting-purple": $supporting-purple
+          "body-secondary": rgba(var(--#{$prefix}secondary-bg-rgb), var(--#{$prefix}bg-opacity)),
+          "body-tertiary": rgba(var(--#{$prefix}tertiary-bg-rgb), var(--#{$prefix}bg-opacity)),
+          "body-emphasis": rgba(var(--#{$prefix}emphasis-bg-rgb), var(--#{$prefix}bg-opacity)),
+          "supporting-green": $supporting-green, // Boosted mod
```

Comment:
They stay in Sass until Orange dark mode right ?

Reply:
Yes that's what I thought for now